packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -96,6 +96,7 @@ class RequestQueue {
    * @private
    */
   async push({request}) {
+    await this._initializationPromise;
```

Comment:
I don't think we can use a getter because this is async, but +1 to all methods that reference `this._queue` should await its initialization.